zucchini/graders/libcheck_grader.py

```diff
@@ -55,6 +56,8 @@ def grade(self, path, grader):
                                    str(grader.test_timeout)},
                               cwd=path, stdout=PIPE, stderr=STDOUT)
 
+        if process.returncode < 0:
+            return self.test_error_grade(strsignal(-process.returncode))
```

Comment:
Did you test and this produces reasonable-looking output? My last couple of PRs have an easy way to test in the PR comments

Reply:
Like we gotta remember that students are going to be staring at this. I trust you if you tested and it looks good tho

Reply:
Honestly I have not yet. I will look at the previous PR comments and test soon (likely tomorrow since I'm probably going to spend the rest of tonight grading quiz 5).

However I have tested strsignal in the interpreter, so unless I made a mistake with this, a segfaulting test should just print "Segmentation fault" to the console.